website/docs/api-reference/props.mdx

```diff
@@ -277,3 +277,10 @@ Called when the active / highlighted cell changes. Called with null when the gri
 > Default: `void`
 
 Called when the selection changes. Called with null when the grid is blurred.
+
+### onColumnsResize
+> Type: `({ widths: Array<number | undefined>) => void`<br />
+> Default: `void`
+
+If provided, the grid will became resizable. This callback is called when the user resizes a column.
```

Comment:
I think we need to have additional boolean flag which enables resizing. One would use resizing without need for handling it.